examples/cfq/models.py

```diff
@@ -147,13 +254,18 @@ def compute_attention_masks(mask_shape: Tuple, lengths: jnp.array):
   mask = mask * (lengths[:, jnp.newaxis] > jnp.arange(mask.shape[1]))
   return mask
 
-
 class Seq2seq(nn.Module):
   """Sequence-to-sequence class using encoder/decoder architecture."""
 
-  def _create_modules(self, hidden_size):
-    encoder = Encoder.partial(hidden_size=hidden_size).shared(name='encoder')
-    decoder = Decoder.shared(name='decoder')
+  def _create_modules(self, hidden_size, num_layers, dropout):
```

Comment:
This can be simplified, I merged this PR yesterday: https://github.com/google/flax/pull/468